unity/Holo/Assets/Editor/ModelPreprocessing/ModelImport/FileSeriesImporter.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+//A class for importing a STL series from a dir
+public class FileSeriesImporter
+{
+    GameObject seriesGameObject = new GameObject();
+    Mesh mesh = new Mesh();
+    private string[] filePaths;

```

Comment:
Either keep all members marked private / public or only public ones. The default visibility level is private if not specified.